tests/data-builders-kotlin/src/test/kotlin/test/DataBuilderTest.kt

```diff
@@ -80,6 +84,20 @@ class DataBuilderTest {
     assertEquals("Rooooaaarr", data.animal.onLion?.roar)
   }
 
+  @Test
+  fun unknownInterfaceChildTest() {
```

Comment:
Why not `unknownInterfaceChildTest` ?

Reply:
Arh naming 😅 - the interface is not unknown, it's its child which is. But for unions this naming doesn't make sense I think (I don't think Cat is a "child" of the Feline union?)

Reply:
I see. I usually refer to `implementation`/`super interface` instead of `child`/`parent` for this kind of stuff. (And `member`/`union`):

* `Cat` is an implementation of the `Animal` interface
* `Animal` is a super interface of `Cat`
* `Cat` is a member of the `Feline` union
* `Feline` is the union containing `Cat`

So for the test I guess `otherInterfaceImplementation()` ?

Reply:
Yes that makes sense! 👍